exchanges/kraken/kraken_wrapper.go

```diff
@@ -282,13 +292,13 @@ func (k *Kraken) UpdateTicker(p currency.Pair, assetType asset.Item) (ticker.Pri
 			}
 
 			tickerPrice = ticker.Price{
-				Last:   v.Last,
-				High:   v.High,
-				Low:    v.Low,
-				Bid:    v.Bid,
-				Ask:    v.Ask,
-				Volume: v.Volume,
-				Open:   v.Open,
+				Last:   tickers[curr].Last,
```

Comment:
For blazing speed, you can create a new variable from the map lookup result rather than looking up it 7 times